### DISCUSSION TITLE
Is there documentation/instructions on installing on a windows machine?

### DISCUSSION THREAD

**Discussion by IntrigueOfMind** [pawelsalawa/sqlitestudio#4237] [Q&A]
Is there documentation/instructions on installing on a windows machine?

Thank you in advance.

**Comment by pawelsalawa** [comment]
Version 3.3.x is not installable. You just download it, unzip it and run it from unzipped folder.
Versions 3.0 and 3.1 had installer. I don't remember about the 3.2.

The version that is currently in development (3.4.0) will again have an installer.

**Comment by IntrigueOfMind** [comment]
Can you please explain further how to "run it from the unzipped folder"? Thank you for your patience.

**Reply by pawelsalawa** [reply]
Once you unzip it, enter the extracted folder (`SQLiteStudio` or `SQLiteStudio-version`) and from there double-click on `SQLiteStudio.exe`.

**Reply by IntrigueOfMind** [reply]
Thanks so much!
